discord/types/stage_instance.py

```diff
@@ -0,0 +1,37 @@
+"""
```

Comment:
Maybe move these typings to `discord.types.channel` instead its own dedicated file? I'm not sure these are special enough from channels to warrant this separation atm.